README.md

```diff
@@ -20,10 +20,10 @@ As a general guideline, please follow the model RDF spec to describe AI models a
 A BioImage.IO-compatible Resource Description File (RDF) is a YAML file with a set of specifically defined fields. 
 
 You can find detailed field definitions here: 
-   - [general RDF spec 0.3](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_v0_3.md)
+   - [general RDF spec 0.2.0](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_v0_2_0.md)
```

Comment:
patch/micro version is actually ignored here on purpose... as it is setup currently only the latest patch version is really documented and has a json schema. there should not be significant/incompatible changes and if the specific patch/micro version is actually required we can go to the appropriate git tag.

Reply:
```suggestion
   - [general RDF spec 0.2.x](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_v0_2.md)
```